pkgs/top-level/all-packages.nix

```diff
@@ -5235,7 +5235,7 @@ with pkgs;
 
   haskell = callPackage ./haskell-packages.nix { };
 
-  haskellPackages = haskell.packages.ghc802.override {
+  haskellPackages = haskell.packages.ghcDefault.override {
```

Comment:
Well, the whole point of this is that there was no „default“ packageset before. What you refer to is the compiler derivation (`haskell.compiler.ghc`).